app/models/refinery/jobs/job.rb

```diff
@@ -36,7 +31,16 @@ def friendly_id_source
         title
       end
 
+      def self.latest(number = 5)
+        limit(number).order('created_at DESC')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.